src/main/java/com/socrata/api/Soda2Producer.java

```diff
@@ -76,6 +78,7 @@ public static final Soda2Producer newProducerWithRequestId(final String url, Str
     public Soda2Producer(HttpLowLevel httpLowLevel)
     {
         super(httpLowLevel);
+        factory = new JsonFactory();
```

Comment:
pushed 5d70349

This is a bit of a struggle because some of the tests are also broken on master so I'm not sure what tests are broken because of my changes are which are just broken. For example, DatesTest#testDateCrud